internal/code-usage-scan-tool/Readme.md

```diff
@@ -0,0 +1,36 @@
+# Automating Multiple Usage Detection
+
+I have implemented the tool as an MSF4J microservice which exposes two endpoints to populate the database and to query the data in the database for a given method name. The method usages are extracted using the ASM Java Bytecode Manipulation and Analysis Framework. The application scans jar files and extracts the details about all the method invocations including the java util methods. All the extracted method references are stored in a database which was implemented as a mongodb database.
```

Comment:
Need to remove this form of "I have implemented"... think from end user's perspective and give a generic description on what the tool is doing.